### PR TITLE
Update hero line heights

### PR DESCRIPTION
**Pull request by Kenneth-Siu**

### CODE REVIEW

**Comment by Kenneth-Siu** [comment]
> a previous PR that was merged a few mins ago "stole" your setup.py version number
> 
> please update to `.2` :)

nooooo D:

File: directory_components/export_elements/sass/partials/_typography.scss [hunk 1]
```diff
@@ -60,15 +61,15 @@ $line-height: 1.6;
   $font-size: 36px;
   $font-size-640: 24px;
   $font-size-print: 18pt;
-  @include _core-font-generator($font-size, $font-size-640, $font-size-print, $line-height, $line-height, $tabular-numbers, $font-weight);
+  @include _core-font-generator($font-size, $font-size-640, $font-size-print, $hero-line-height, $hero-line-height, $tabular-numbers, $font-weight);
```
Comment (eadpearce) [inline directory_components/export_elements/sass/partials/_typography.scss]: should this var be passed in twice?

Comment (Kenneth-Siu) [reply directory_components/export_elements/sass/partials/_typography.scss]: Yeah, it needs a line height for small screens and large screens.

Comment (eadpearce) [reply directory_components/export_elements/sass/partials/_typography.scss]: ooh i see

**Pull request merged**